src/main/java/edu/ucsb/cs56/ucsbapi/academics/curriculums/v1/classes/GeneralEducation.java

```diff
@@ -0,0 +1,10 @@
+package edu.ucsb.cs56.ucsbapi.academics.curriculums.v1.classes;
+
+import lombok.Data;
+
+@Data
+public class GeneralEducation {
+    public String geCode;
+    public String geCollege;
+    public GeneralEducation() {}
+}
```

Comment:
We're actually trying to move away from using Lombok. Can you remove the `lombok.Data` import/annotation and instead generate getters and setters for your fields? VS Code has a nice feature that will automatically generate those for you.